Packs/Yara/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,16 @@
+#### Scripts
+
+##### YaraScan
+
+- Updated the script to be able to handle multiple yara rules following the format of the yara module 
```

Comment:
no need for a detailed description - please keep it high-level and more to the point.